views/dashboard.hbs

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Dashboard</title>
+    <!-- <link href="style1.css" rel="stylesheet" media="all" /> -->
+		<link rel = "stylesheet" href = "style1.css">              
+		<!-- bootstrap and jquery -->
+                
+        <!-- Latest compiled and minified CSS -->
+        <link rel = "stylesheet" href = "https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" />
+        <!-- <link rel="stylesheet" href="/bower_components/bootstrap/dist/css/bootstrap.min.css" /> -->
+            
+        <!-- font awesome -->
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" />
+
+        <!-- Google font  -->
+		<link href="https://fonts.googleapis.com/css?family=Jua|Slabo+27px" rel="stylesheet">
+		
+        <!-- jQuery library -->
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+
+        <!-- Latest compiled JavaScript -->
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>	
+		
+</head>
+<body class="container-fluid">
+    <div class="contentBox" style="">
+        <div id="toolbar" >
+            <!-- Row 1-->
+            <!-- For some reason the BGcolor was not working from CSS side i will need help here -->
+            <div class="row toolbar-parent" style="background-color: white">
```

Comment:
avoid inline styles for better page performance

Reply:
I did try to avoid it but for some reason, bootstrap was overriding my CSS class colour 
but taking its an initial mockup. 

so sorry about that i had to try.

Reply:
@jaibeer72 try **!important** you'll override bootstrap, btw we are using materialize not bootstrap for this project.

Reply:
@mepsrajput 
All right i will redo it with material at 7pm today i will start mostly by 9pm i should have a mock-up